core/src/utils/native/capacitor.ts

```diff
@@ -0,0 +1,13 @@
+import type { CapacitorGlobal } from '@capacitor/core';
+
+let capacitor: CapacitorGlobal | undefined;
```

Comment:
If we are querying `window.Capacitor` do we even need to cache the value here?

Reply:
Removed here: https://github.com/ionic-team/ionic-framework/pull/27755/commits/ad7c02c2d5e5e91e5cb957b213c0b166c34fc6b9 was probably a premature optimization.